package.json

```diff
@@ -78,8 +79,10 @@
     "fixtures:load": "cozy-fixtures load tests/fixtures",
     "fixtures:loader": "coffee tests/fixtures/loader.coffee",
     "lint": "npm-run-all 'lint:*'",
-    "lint:coffeelint": "coffeelint --file coffeelint.json --quiet ./server/**/*.coffee",
-    "lint:jshint": "coffee-jshint --options node ./server/**/*.coffee",
+    "lint:server": "npm-run-all 'lint:server:*:*'",
+    "lint:server:code:coffeelint": "./node_modules/.bin/coffeelint --file coffeelint.json  server.coffee ./server/*.coffee ./server/**/*.coffee",
```

Comment:
Yup, npm script command first search in _node_modules_ to find executable, and then fallback to global env if it doesn't find it.